corehq/apps/smsbillables/models.py

```diff
@@ -353,3 +373,41 @@ def create(cls, message_log, api_response=None):
         billable.save()
 
         return billable
+
+    @classmethod
+    def _get_twilio_charges(cls, billable, message_log, backend_instance):
+        def _get_twilio_client(twilio_backend_id):
+            twilio_backend = SQLMobileBackend.get(twilio_backend_id)
```

Comment:
this should be `SQLMobileBackend.load(twilio_backend_id, api_id=SQLTwilioBackend.get_api_id(), is_couch_id=True, include_deleted=True)`